DatabaseBuddy.Core/Extender/NumericExtender.cs

```diff
@@ -16,33 +16,12 @@ public static long ToLongValue(this object Value, long DefaultValue)
       }
       catch { return DefaultValue; }
     }
-
-    public static long ToLongValue(this long Value)
-    {
-      return Value;
-    }
-
-    public static long ToLongValue(this long? Value)
```

Comment:
The ToLongValue with a nullable long is a valid method and should not be removed because this could be needed quite often.